src/System.Linq.Expressions/src/System/Linq/Expressions/DynamicExpression.cs

```diff
@@ -436,7 +436,7 @@ public static DynamicExpression MakeDynamic(Type delegateType, CallSiteBinder bi
         /// <see cref="DynamicExpression.Binder">Binder</see>, and
         /// <see cref="DynamicExpression.Arguments">Arguments</see> set to the specified values.
         /// </returns>
-        public static DynamicExpression MakeDynamic(Type delegateType, CallSiteBinder binder, Expression arg0, Expression arg1, Expression arg2)
+        public static new DynamicExpression MakeDynamic(Type delegateType, CallSiteBinder binder, Expression arg0, Expression arg1, Expression arg2)
```

Comment:
Because this methods are overriding the ones I added in this PR for its base class (Expression). This is the way we have their implementation in Desktop as well.